org.beeref.BeeRef.json

```
@@ -0,0 +1,51 @@
+{
+    "id": "org.beeref.BeeRef",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.3",
+    "sdk": "org.kde.Sdk",
+    "base": "com.riverbankcomputing.PyQt.BaseApp",
+    "base-version": "6.3",
+    "command": "beeref",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=network",
+        "--socket=x11",
```

```suggestion
        "--socket=fallback-x11",
```
prefer wayland where available